app/models/user_subject_queue.rb

```diff
@@ -22,17 +22,17 @@ def self.scope_for(action, groups, opts={})
 
   def self.enqueue_subject_for_user(user: nil, workflow: nil, set_member_subject: nil)
```

Comment:
Let's keep this in mind for the future. The administrative functions in a our clients will definitely need to know about and handle set_member_subjects. I didn't think it made sense to make a client that was only interested in submitting classifications worry about them.